build.gradle

```diff
@@ -6,4 +6,12 @@ allprojects {
     repositories {
         mavenCentral()
     }
+}
+
+subprojects {
+    tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
```

Comment:
I've checked `compileKotlin` and `compileTestKotlin` have type `org.jetbrains.kotlin.gradle.tasks.KotlinCompile`, so I've written it this way.
If it matters to use `org.jetbrains.kotlin.gradle.dsl.KotlinJvmCompile` type I'll change it.